test/selenium/testlib_avocado/seleniumlib.py

```diff
@@ -307,7 +293,7 @@ def select_by_text(self, element, value):
     def select_by_value(self, element, value):
         return self.select(element=element, select_function="select_by_value", value=value)
 
-    def wait(self, method, text, baseelement, overridetry, fatal, cond, jscheck, text_):
+    def wait(self, method, text, baseelement, overridetry, fatal, cond, text_, wait_data_loaded=False):
```

Comment:
I hope, that nobody uses positional parameters for this, but suggest to switch positions of ``text_, wait_data_loaded=False`` with this generic method, and no not pass default param, as this should be, but I understand that  the reason is to not pass this default to each specific functiono

Reply:
Ack, I changed it.